brion/plugin/compartmentReportBinary.cpp

```diff
@@ -706,23 +716,23 @@ bool CompartmentReportBinary::_parseMapping()
                 const uint64_t frameIndex =
                     j + ((info.dataOffset - _header.dataBlockOffset) /
                          sizeof(float));
```

Comment:
It is, but since all the machines we use are IEEE 745 I guess it's impossible to find something different from 4. In any case, I prefer keeping sizeof(float) for consistency with the rest of code style.